src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -572,6 +572,25 @@ public function text()
     }
 
     /**
+     * Returns the flattened node value of the first node of the list.
+     *
+     * Surrounding spaces/newlines are removed,
+     * and consecutive spaces/newlines are reduced to one space.
+     *
+     * @return string The flattened node value
+     *
+     * @throws \InvalidArgumentException When current node is empty
+     */
+    public function flatText()
```

Comment:
Here it's more a getter than an action, so I think a noun rather than a verb is fine (and actually better).

By the way, I had thought about `flattenedText`, but it's too long.